packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss

```diff
@@ -29,13 +29,29 @@
         padding-right: 1rem;
     }
     &-card-container {
+        position: relative;
+        overflow: hidden;
         width: 27rem;
         border: 1px solid var(--general-hover);
-        padding: 2.4rem;
         border-radius: 2.4rem;
         &:hover {
             box-shadow: 0 2px 8px 0 var(--shadow-menu);
         }
+        &__banner {
+            position: absolute;
+            z-index: 1000;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            padding: 1.5rem;
+            width: 13rem;
+            height: 2rem;
+            right: -4rem;
```

Comment:
we usually dont use the right left in (-ve) kindly use other css property 
Its a vague value